framework/test/unit/modules/token/stores/supported_tokens.spec.ts

```diff
@@ -58,6 +58,8 @@ describe('SupportedTokensStore', () => {
 		});
 
 		it('should return true if tokenID is LSK', async () => {
+			await store.set(context, Buffer.from([1, 0, 0, 0]), { supportedTokenIDs: [] });
```

Comment:
we should add a test where it's different token from mainchain. in this example, something like `[1,0,0,0,0,0,0,1]`